api-tests/checkout-tests/checkout-api-dev.tests.json

```diff
@@ -187,9 +187,11 @@
 					"listen": "test",
 					"script": {
 						"exec": [
+							"var jsonData = JSON.parse(responseBody);",
+							"postman.setEnvironmentVariable(\"TRANSACTION_ID\", jsonData.transactionId);",
+							"postman.setEnvironmentVariable(\"AUTH_TOKEN\", jsonData.authToken);",
```

Comment:
Move these **after** checking if the response is OK, otherwise the tests would overwrite valid values with invalid ones